docs/examples/config/example.env

```diff
@@ -287,7 +287,8 @@
 
 # PL_PROVISIONING_BACKEND=directory
 # PL_DIRECTORY_PROVIDERS=scim
-# PL_DIRECTORY_SCIM_PORT=[required]
+# PL_DIRECTORY_SCIM_URL=http://localhost:5000
```

Comment:
We _might_ want to consider eventually not requiring this one and just use the `serverScimServerUrl` isntead, but it seemed out-of-scope to pass the server config in as well, and it's not terrible to have two separate variables that need to have the same value (even if not ideal).